content/USER_GUIDE.md

```diff
@@ -0,0 +1,242 @@
+# **FLASHDeconv & FLASHTnT User Guide**
+
+Welcome to the **FLASHDeconv & FLASHTnT User Guide**. This guide provides a step-by-step walkthrough on using **FLASHDeconv** and **FLASHTnT**, from uploading data to processing and viewing results.
+
+---
+
+## **1️⃣ Uploading MS Data**
+
+1. Navigate to **FLASHDeconv > Workflow** in the sidebar.
+2. Click **"File Upload"** to upload your `.mzML` file.
+3. **Two options to add files:**
+   - **Drag and Drop** your file into the upload box.
+   - **Browse Files** to manually select the `.mzML` file.
+4. Click **"Add MS Data"** to confirm the upload.
+
+
+![Configure Parameters](/static/Images/flashdeconv_upload.png)
+
+---
+
+## **2️⃣ Configuring Parameters**
+
+1. Click the **"Configure"** tab.
+2. Select your uploaded **mzML file** (should appear in the file list).
+3. Adjust **parameters**:
+   - Set **threads** (recommended: 8).
+   - Choose **General Settings** (e.g., `keep_empty_out`, `write_detail`).
+   - Configure **FD settings** like `report_FDR`, `merging_method`, `min_mass`, `max_mass`, `min_charge`.
+   - Adjust **SD settings** for deconvolution accuracy.
+4. If you want to know more about each parameter go to this link: https://openms.de/FLASHDeconv
+
+ 
+![Configure Parameters](/static/Images/flashdeconv_configure.png)
+
+---
+
+## **3️⃣ Running the Workflow**
+
+1. Click on the **"Run"** tab.
+2. Set **log details** to `minimal` (or another level as needed).
+3. Click **"Start Workflow"** to begin the deconvolution process.
+4. Monitor the **log output** to track progress.
+
+  
+![Run Workflow](/static/Images/flashdeconv_run.png)
+
+---
+
+## **4️⃣ Viewing Results**
+
+1. Once the workflow is finished, check the **log messages**.
+2. Navigate to the **"Viewer"** tab in the sidebar to analyze the deconvoluted data.
+3. If needed, **download results** by clicking **"Download Files"**.
+   (Will be explained later in step 7 and 8 in this guide)
+
+---
+
+## **5️⃣ Manual Result Upload**
+
+1. Click on the **"Manual Result Upload"** tab.
+2. Upload FLASHDeconv output files (`*_annotated.mzML` & `*_deconv.mzML`) or TSV files for ECDF Plot.
+3. Browse files or **drag and drop** them into the upload section.
+4. Click **"Add files to workspace"** to finalize.
+
+
+![Manual Upload](/static/Images/flashdeconv_manual_upload.png)
+
+---
+
+## **6️⃣ Using Example Data**
+
+1. Click the **"Example Data"** tab.
+2. Click **"Load Example Data"** to use the preloaded dataset.
+3. The example data will appear in the uploaded experiments table.
+
+ 
+![Example Data](/static/Images/flashdeconv_example_data.png)
+
+---
+
+## **7️⃣ Layout Manager**
+
+The **Layout Manager** allows users to customize the experiment display settings.
+
+1. Select the **number of experiments** to view at once.
+2. Click **"Select..."** to choose components to add:
+   - **MS1 raw heatmap**
```

Comment:
For the components a description would be nice. Maybe somehow hidden/visually separated. I am thinking one sentence for what does it display and one sentence about what it is good for.